health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -137,29 +116,30 @@ const Hypothesis = ({ category, assumptions:initialAssumptions, customProps, onS
                                           id={index}
                                           onChange={(event) => {
-                                            handleAssumptionChange(event, item);
+                                            
+                                            handleAssumptionChange(category,event, item);
+                                          
                                           }}
                                             />
                                         <div className="delete-button">
                                           <DeleteIconv2 />
-                                          <span  style={{color:"red",textDecoration:"Underline" }} onClick={()=> handleDeleteClick(index)}>Delete</span>
+                                          <span  style={{color:"red",textDecoration:"Underline" }} onClick={()=> handleDeleteClick(index)}>{t("DELETE")}</span>
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)